README.md

```diff
@@ -54,7 +82,7 @@ import AnvilSignatureFrame from '@anvilco/react-signature-frame'
 ```
 
 
-## AnvilSignatureModal
+## (Deprecated) AnvilSignatureModal
```

Comment:
This shouldnt be deprecated yet. It can eventually use the `AnvilEmbedFrame` tho